src/identity-providers/add/DetailSettings.tsx

```diff
@@ -95,11 +98,19 @@ export const DetailSettings = () => {
   const { realm } = useRealm();
 
   useFetch(
-    () => adminClient.identityProviders.findOne({ alias: id }),
+    () => {
+      return Promise.all([
+        adminClient.identityProviders.findOne({ alias: id }),
+        adminClient.identityProviders.findMappers({ alias: id }),
+        adminClient.identityProviders.findMapperTypes({ alias: id }),
+      ]);
+    },
```

Comment:
Since this is an arrow function you can return the value implicitly to shorten this function.
```suggestion
    () =>
      Promise.all([
        adminClient.identityProviders.findOne({ alias: id }),
        adminClient.identityProviders.findMappers({ alias: id }),
        adminClient.identityProviders.findMapperTypes({ alias: id }),
      ]),
```